### PR TITLE
Fix Flows/Flores for NNCs

### PR DESCRIPTION
**Pull request by totto82**
Restores NNC output for Flows. 

See #873

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

File: opm/models/discretization/common/tpfalinearizer.hh [hunk 1]
```diff
@@ -673,30 +674,35 @@ public:
...
             const IntensiveQuantities& insideIntQuants = model_().intensiveQuantities(globI, /*timeIdx*/ 0);
             LocalResidual::computeBoundaryFlux(adres, problem_(), bdyInfo.bcdata, insideIntQuants, globI);
             adres *= bdyInfo.bcdata.faceArea;
-
+            const unsigned bfIndex = bdyInfo.bfIndex;
             if (enableFlows) {
-                for (unsigned phaseIdx = 0; phaseIdx < numEq; ++ phaseIdx) {
-                    flowsInfo_[globI][bdyInfo.dir].flow[phaseIdx] = adres[phaseIdx].value();
+                for (unsigned eqIdx = 0; eqIdx < numEq; ++ eqIdx) {
+                    flowsInfo_[globI][nbInfos.size() + bfIndex].flow[eqIdx] = adres[eqIdx].value();
```
Comment (atgeirr) [inline opm/models/discretization/common/tpfalinearizer.hh:705]: To check I understood this correctly: this assumes that for every cell, flowsInfo_ stores a number of objects equal to the number of neighbours plus the number of boundaries, and the flows for boundaries come last.

**Review by atgeirr — COMMENTED** [review]
Looks mostly fine, a few minor issues to address before merging.

File: opm/models/discretization/common/tpfalinearizer.hh (hunk 1 above)
Comment (totto82) [reply opm/models/discretization/common/tpfalinearizer.hh:705]: Yes. That is correct. That is also why I only resize to (numCells, 6) i.e guessing on 6 neighbors. (your comment above)

Comment (atgeirr) [reply opm/models/discretization/common/tpfalinearizer.hh:705]: It is a reserve not a resize, but apart from that I still do not understand. A typical grid with ~1000 cells will have ~6000 neighbours, so reserving just space for 6 in the data array will ensure that the data array gets reallocated many times. How does this relate to the boundaries being added as well? If anything, that would increase the required space.

File: opm/models/discretization/common/tpfalinearizer.hh [hunk 2]
```diff
@@ -565,13 +565,13 @@ private:
         }
 
         if (anyFlows) {
-            flowsInfo_.reserve(numCells, 6 * numCells);
+            flowsInfo_.reserve(numCells, 6);
```
[the remark this answers is not in the document]
Comment (totto82) [reply opm/models/discretization/common/tpfalinearizer.hh:568]: Thanks. I misread the SparseTable class. Will revert.

**Comment by totto82** [comment]
jenkins build this please

**Comment by totto82** [comment]
jenkins build this please

**Pull request merged**